src/content/community/language-resources/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Language resources
+description: Non-english resources to learn about Ethereum
+sidebar: true
+lang: en
+---
+
+# Language resources {#language-resources}
+
+The Ethereum community is global and comprised of millions of non-English speakers.
+
+Our aim is to provide educational content in all languages and help overcome the language barriers that make onboarding people from all over the world to Ethereum a challenge.
+
+If you prefer reading in your native language or know someone who doesn’t speak English, you can find a list of useful non-English resources below.
+
+Know of an educational resource in your language? [Open an issue](https://github.com/ethereum/ethereum-org-website/issues/new/choose) to add it to the list!
+
+If you are bilingual and want to help us reach more people, you can also get involved with the [ethereum.org Translation Program](https://ethereum.org/en/contributing/translation-program/#translation-program) and help us translate the website.
+
+Hundreds of thousands of Ethereum enthusiasts gather in these online forums to share news, talk about recent developments, debate technical issues, and imagine the future.
+
+## Community resources {#community}
```

Comment:
Related to my comment above, perhaps we add a H2 section about this on ethereum.org resources / languages, where we mention that we have a bunch of on-site translations.

Reply:
@samajammin Clarifying, are you suggesting adding a section to the `/translations` page, calling this page out?

Reply:
@wackerow I think that's a good idea too! So I'd request 2 changes.

1. Add an H2 on this page - "Ethereum.org resources" that explains our translation program & that we translate all our content into many langues, with a link to our /languages page.
2. Add a section on the /languages page the mentions & links to this page (e.g. "In addition to translating ethereum.org content, we also maintain a curated list of language resources...").